openwisp_notifications/base/models.py

```diff
@@ -86,6 +119,28 @@ def email_subject(self):
         else:
             return self.message
 
+    def _related_object(self, field):
+        obj_id = getattr(self, f'{field}_object_id')
+        if obj_id:
```

Comment:
Please use this form to avoid nesting the rest of the code and hence improving readability:
```python
if not obj_id:
    return
```